codes/c/chapter_sorting/bubble_sort.c

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file bubble_sort.c
```

Comment:
Please make the format of the file header match C++.

Reply:
OK. Fixed it.